packages/e2e/test/local-network/local-network.test.ts

```diff
@@ -0,0 +1,117 @@
+import * as envalid from 'envalid';
+import { Cardano } from '@cardano-sdk/core';
+import { FaucetProvider } from '../../src/FaucetProvider';
+import { KeyManagement, SingleAddressWallet } from '@cardano-sdk/wallet';
+import { faucetProviderFactory, getLogger, getWallet } from '../../src/factories';
+import { filter, firstValueFrom, map } from 'rxjs';
+
+// Verify environment.
+export const env = envalid.cleanEnv(process.env, {
+  ASSET_PROVIDER: envalid.str(),
+  ASSET_PROVIDER_PARAMS: envalid.json({ default: {} }),
+  CHAIN_HISTORY_PROVIDER: envalid.str(),
+  CHAIN_HISTORY_PROVIDER_PARAMS: envalid.json({ default: {} }),
+  FAUCET_PROVIDER: envalid.str(),
+  FAUCET_PROVIDER_PARAMS: envalid.json({ default: {} }),
+  KEY_MANAGEMENT_PARAMS: envalid.json({ default: {} }),
+  KEY_MANAGEMENT_PROVIDER: envalid.str(),
+  LOGGER_MIN_SEVERITY: envalid.str({ default: 'info' }),
+  NETWORK_INFO_PROVIDER: envalid.str(),
+  NETWORK_INFO_PROVIDER_PARAMS: envalid.json({ default: {} }),
+  REWARDS_PROVIDER: envalid.str(),
+  REWARDS_PROVIDER_PARAMS: envalid.json({ default: {} }),
+  STAKE_POOL_PROVIDER: envalid.str(),
+  STAKE_POOL_PROVIDER_PARAMS: envalid.json({ default: {} }),
+  TX_SUBMIT_PROVIDER: envalid.str(),
+  TX_SUBMIT_PROVIDER_PARAMS: envalid.json({ default: {} }),
+  UTXO_PROVIDER: envalid.str(),
+  UTXO_PROVIDER_PARAMS: envalid.json({ default: {} })
+});
+
+const logger = getLogger(env.LOGGER_MIN_SEVERITY);
+
+describe('Local Network', () => {
+  let _faucetProvider: FaucetProvider;
+
+  beforeAll(async () => {
+    _faucetProvider = await faucetProviderFactory.create(env.FAUCET_PROVIDER, env.FAUCET_PROVIDER_PARAMS);
```

Comment:
```suggestion
    faucetProvider = await faucetProviderFactory.create(env.FAUCET_PROVIDER, env.FAUCET_PROVIDER_PARAMS);
```

We're not using this convention anywhere else (that I know of).

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/342/commits/e802292cef7025c7b651ceac8978f4afd8b3f02f